sql/item_mods.sql

```diff
@@ -3573,6 +3573,12 @@ INSERT INTO `item_mods` VALUES (10846,23,5);    -- ATT: 5
 INSERT INTO `item_mods` VALUES (10846,73,3);    -- STORETP: 3
 INSERT INTO `item_mods` VALUES (10846,384,600); -- HASTE_GEAR: 600
 
+-- Orc Belt (10847)
+-- TODO: Enchantment: Costume
+
+-- Quadav Belt (10848)
+-- TODO: Enchantment: Costume
```

Comment:
Are you just needing the costume ID or wanting to avoid doing script in same PR as the SQL work? I would suggest not adding these to the file until implementing the script. The placeholder isn't worth adding more TODOs.

Reply:
Removed for now. Will dig into it more later.